tests/integration/cartography/intel/lastpass/test_users.py

```diff
@@ -0,0 +1,38 @@
+import cartography.intel.lastpass.users
+import tests.data.lastpass.users
+
+
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_lastpass_users(neo4j_session):
+
```

Comment:
As one more nitpicky suggestion, can you comment the test with 

```python
# Arrange
...

# Act
sync(...)

# Assert
```

As mentioned on Slack, we will use this PR to be the main example that we use for new contributors to copy so I am being a bit extra right now :)